server/src/main/kotlin/suwayomi/tachidesk/graphql/mutations/UpdateMutation.kt

```diff
@@ -0,0 +1,69 @@
+package suwayomi.tachidesk.graphql.mutations
+
+import org.jetbrains.exposed.sql.select
+import org.jetbrains.exposed.sql.transactions.transaction
+import org.kodein.di.DI
+import org.kodein.di.conf.global
+import org.kodein.di.instance
+import suwayomi.tachidesk.graphql.types.UpdateStatus
+import suwayomi.tachidesk.manga.impl.Category
+import suwayomi.tachidesk.manga.impl.update.IUpdater
+import suwayomi.tachidesk.manga.model.table.CategoryTable
+import suwayomi.tachidesk.manga.model.table.toDataClass
+
+class UpdateMutation {
+    private val updater by DI.global.instance<IUpdater>()
+
+    data class UpdateGlobalMangaInput(
```

Comment:
would it also be better to use `mangas` instead of `manga` to make it clear that it's plural?

Reply:
Manga rolls off the tongue better, its not like we need to be specific here